app/src/main/java/dev/msfjarvis/aps/util/auth/BiometricAuthenticator.kt

```diff
@@ -45,6 +63,7 @@ object BiometricAuthenticator {
           logcat(TAG) { "BiometricAuthentication error: errorCode=$errorCode, msg=$errString" }
           callback(
             when (errorCode) {
+              BiometricPrompt.ERROR_UNABLE_TO_PROCESS,
```

Comment:
I think `UNABLE_TO_PROCESS` should return retry 🤔 
Here's what the docs mention
```
The sensor was unable to process the current image.
```

Reply:
Good point, made the change.